src/Uno.UWP/Media/SpeechRecognition/SpeechRecognizerState.cs

```diff
@@ -0,0 +1,13 @@
+namespace Windows.Media.SpeechRecognition
+{
+	public enum SpeechRecognizerState 
```

Comment:
Again are all of these actually supported?

Reply:
Will do